tests/browser-automation/pageoperator.js

```diff
@@ -0,0 +1,59 @@
+const { BrowserPageWidths } = require("./constants");
+
+/**
+ * Responsible for browser navigation based on given test locations.
+ * Each test location is a JSON object which may contains:
```

Comment:
nit: It looks like they are regular javascript objects and not JSON, so I would say: "... is an object which may contain:"